.vib/golang/goss/golang.yaml

```diff
@@ -0,0 +1,13 @@
+command:
+  check-app-version:
+    exec: go version | grep -q $(echo $APP_VERSION | cut -d '.' -f 1,2)
```

Comment:
This is not completely valid. Imagine we are testing go `1.20.2`:

```bash
$ echo "1.20.2" | cut -d '.' -f 1,2
1.20
```

But for whatever reason, this release was not properly compiled and it is shipping `1.20.1`. The test will report a false positive:

```bash
$ echo "go version go1.20.1 linux/amd64" | grep -q $(echo "1.20.2" | cut -d '.' -f 1,2)
$ echo $?
0
```

Reply:
You are right, I totally missed this. Changed now!